homeassistant/components/tuya/light.py

```diff
@@ -110,8 +110,9 @@ class TuyaLightEntityDescription(LightEntityDescription):
         ),
         # Not documented
         # Based on multiple reports: manufacturer customized Dimmer 2 switches
+        # Second Light is not SWITCH_LED_1, it is SWITCH_1
```

Comment:
This comment is not needed, as we don't need to record change history in code (Git is providing that already)

```suggestion
```

Reply:
Thank you.